### DISCUSSION TITLE
Docs: Add examples/explanation of the possible `ruleset` objects

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#88] [Feature Requests]
Edit: This is a misinformed issue, sorry for the confusion! See follow up comment for context

This API Reference docs for the `ruleset` objects but describe that it will be an object, but don't break down the schema: https://docs.battlesnake.com/references/api#game

I think it would be valuable to describe the schema of the different objects that are sent depending on the game mode. Specifically I think this would help with Royale, and specifically the Royale Variants like 'Hot Sauce' and 'Quick Sauce' that made an appearance on the Snake Pit Live stream today.

Right now the Royale ruleset object takes the form of
```
"ruleset":{"damagePerTurn":"15","foodSpawnChance":"20","minimumFood":"1","name":"royale","shrinkEveryNTurns":"20", "version": "1"}
```

Of particular interest is the `damagePerTurn` and `shrinkEveryNTurns` parameters, which correspond to the two variants I mentioned. These aren't super discoverable unless you extract the full JSON sent to your snake and look for things you don't recognize. It would be great to get them documented!

Thanks!

**Comment by coreyja** [comment]
Ohhhh, so I might have been wrong about these actually being included already.

I see them because I use the Game Downloader Firefox extension, which in turn uses the `/games/` API endpoint. Ex: https://engine.battlesnake.com/games/079c11b1-da87-4dd5-84b1-014a59a91048

But it seems from Discord that this is NOT the same ruleset that is actually sent to the snakes. So this probably isn't really a **documentation issue**, but I will go thumbs up #75 cause it would be great to get these in actual game play!

**Comment by bvanvugt** [comment]
@coreyja awesome, thanks for bringing this up anyway. At some point we'd like to add these params to the official API, but likely in a different / documented format.

Closing discussion as won't fix :)
